cmd/executor/gh/main.go

```diff
@@ -98,6 +99,11 @@ func (e *GHExecutor) Execute(ctx context.Context, in executor.ExecuteInput) (exe
 	}, nil
 }
 
+// Help returns help message
+func (*GHExecutor) Help(ctx context.Context) (interactive.Message, error) {
+	return interactive.Message{}, nil
```

Comment:
Please add help based on https://blog-post.botkube-docs.pages.dev/blog/custom-executors/

![image](https://user-images.githubusercontent.com/7155799/215517257-91130bca-45a9-4f42-b6b5-b1b7406215c4.png)

This is a low-hanging fruit, so.. 🙂 


```
Usage: `gh create issue KIND/NAME [-n, --namespace]`
```
Will be totally fine 👍